cmake/cc_rules.cmake

```diff
@@ -93,13 +94,15 @@ endfunction()
 #   target
 #   SOURCES sources...
 #   DEPENDS libraries...
+#   [EXCLUDE_FROM_ALL]
```

Comment:
This seems unrelated?

(No objection to it though.)

Reply:
It was pass one at making this happen. Turned out to not be enough but didn't seem harmful.